<?xml version='1.0'?>
<launch>
	<node pkg="ml_detector" type="ml_generate_boards" name="ml_generate_boards" clear_params="true" output="screen">
		<param name="output_directory" value="$(find ml_detector)/img/" type="str"/>

		<rosparam command="load" file="$(find ml_detector)/launch/board_config.yaml"/>
	</node>
</launch>
